controllers/iperf3/serverdeployment.go

```diff
@@ -73,8 +73,9 @@ func NewServerDeployment(cr *perfv1alpha1.Iperf3) *appsv1.Deployment {
 
 	deployment := appsv1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      serverDeploymentName(cr),
-			Namespace: cr.Namespace,
+			Name:        serverDeploymentName(cr),
+			Namespace:   cr.Namespace,
+			Annotations: cr.Spec.ServerConfiguration.PodConfigurationSpec.PodAnnotations,
```

Comment:
In this case the name `PodAnnotations` is quite misleading. You're also using the map to annotate the deployment, which I think is fine, but then it should be named simply `Annotations`.

Reply:
I agree with this. i was following the formatting of the other sections in the `PodConfigurationSpec` such as `PodLabels` or `PodScheduling`.

Reply:
@julio-santiesteban can you confirm you would like me to change the naming?

Reply:
`PodLabels` and `PodScheduling` only applies to the pod. In contrast, you apply the given annotations to the deployment / job, too, so yes, I do think simply `Annotations` would be a better name.

Reply:
@julio-santiesteban i have updated the branch and changed the value to Annotations